src/Jackalope/Version/VersionHandler.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Jackalope\Version;
+
+use Jackalope\Node;
+use Jackalope\Transport\TransportInterface;
+use PHPCR\UnsupportedRepositoryOperationException;
+
+class VersionHandler
+{
+    const MIX_VERSIONABLE = 'mix:versionable';
+    const MIX_SIMPLE_VERSIONABLE = 'mix:simpleVersionable';
```

Comment:
i am quite uncertain. it would make most sense probably to translate the constant programmatically, with the current namespace registry. (there is a phpcr feature called local namespace remapping, so theoretically a user could remap `mix` to something else and all our code would break).

@dantleech do you have an idea for this? should we change Node::isNodeType so that it handles the long form of node type names too? sounds like the least surprising way of doing things.